common/config/kafkaConfig.go

```diff
@@ -43,7 +43,8 @@ type (
 
 	// TopicConfig describes the mapping from topic to Kafka cluster
 	TopicConfig struct {
-		Cluster string `yaml:"cluster"`
+		Cluster  string `yaml:"cluster"`
+		IsSecure *bool  `yaml:"isSecure,omitempty"`
```

Comment:
We don't need to make it a pointer, since it's false by default. Personally, I think it's annoying to deal with pointers.

Reply:
bool pointers are useful if you need to differentiate between unset or not. Otherwise just two regular bools with 2 state is preferred.